script/fetch-issues.js

```diff
@@ -223,24 +295,52 @@ function generateRepoInfo(owner, names, issues) {
   return result;
 }
 
+function filterRecent(issues) {
+  // only keep issues created/updated in the past month
```

Comment:
I'm still not sure how this generated data behaves on gh-board, "updated in the past month" means we'll get holes in between, e.g. issues which are created 2 months ago that never got updated won't show here, but issues created 3 months ago that just got updated will show here.

How does the UI behaves? does it fetches the hole in between? or does nothing?

Reply:
It doesn't fetch the hole in between. It only fetches issues/prs that are updated after `repo. lastSeenAt`.

ref. https://github.com/coala/gh-board/blob/60b8a10f123d2bec71c22c31f3e281ad9609fa07/src/issue-store.js#L198

Is that a problem? It's the expected behavior IMO. 

`issues which are created 2 months ago that never got updated won't show here`

That means those issues are probably stale or invalid.

` issues created 3 months ago that just got updated will show here`

That means those issues are being discussed or there's work under it, recently.

Reply:
I'm fine with it as long as it doesn't break.